### DISCUSSION TITLE
【求解答】有关游戏原画师/设计师

### DISCUSSION THREAD

**Discussion by hahaashbin** [share121/inter-knot#5846] [公告]
有没有绳匠的目标是做游戏原画/设计师（主要是人物）
或者已经在游戏公司里工作

能否分享下如果想要进游戏公司原画/设计岗位所需要的技能和大概的水平需要到什么程度。

**Comment by MolandX** [comment]
超纲了啊喂，但以路人身份揣测一下进大厂一般都是各大美院专业对口毕业生叭🤔
